t/16-markdown.t

```diff
@@ -66,6 +66,16 @@ subtest 'bugrefs' => sub {
 subtest 'openQA additions' => sub {
     is markdown_to_html('https://example.com'),
       qq{<p><a href="https://example.com">https://example.com</a></p>\n}, 'URL turned into a link';
+    is markdown_to_html('https://example.com/#fragment_-'),
+      qq{<p><a href="https://example.com/#fragment_-">https://example.com/#fragment_-</a></p>\n},
+      'URL with fragment turned into a link';
+    is markdown_to_html('https://example.com/#fragment/<script>test</script>'),
+qq{<p><a href="https://example.com/#fragment/">https://example.com/#fragment/</a><!-- raw HTML omitted -->test<!-- raw HTML omitted --></p>\n},
+      'URL with fragment + script turned into a link';
+    is markdown_to_html('https://example.com/#?(.-/\''),
```

Comment:
I suggest to extend this, e.g. write here

```suggestion
    is markdown_to_html('https://example.com/#?(.-/\' and some text'),
```

and ensure that ` and some text` will not become part of the link

Reply:
Thanks for the suggestion, implemented